tests/returntypes/test_parameter_shift_new.py

```diff
@@ -967,7 +967,7 @@ def test_variance_gradients_agree_finite_differences(self, tol):
     def test_fallback(self, mocker, tol):
         """Test that fallback gradient functions are correctly used"""
         spy = mocker.spy(qml.gradients, "finite_diff")
-        dev = qml.device("default.qubit.autograd", wires=2)
+        dev = qml.device("default.qubit", wires=3)
```

Comment:
This change here was merely done to be able to distinguish the axes in the results with having 3 measurements and 2 trainable params (instead of both being 2).

Reply:
Great thank you!